src/PageLayout/PageLayout.stories.tsx

```diff
@@ -0,0 +1,298 @@
+    table: {
+      category: 'Header',
+      defaultValue: {
+        summary: '"inherit"'
+      }
+    }
+  },
+  'Content.width': {
+    type: {
+      name: 'enum',
+      value: ['full', 'medium', 'large', 'xlarge']
+    },
+    defaultValue: 'full',
+    control: {
+      type: 'radio'
+    },
+    table: {
+      category: 'Content',
+      defaultValue: {
+        summary: '"full"'
+      }
+    }
+  },
+  'Show pane?': {
+    type: 'boolean',
+    defaultValue: true,
+    table: {
+      category: 'Pane'
+    }
+  },
+  'Pane.position': {
+    type: {
+      name: 'enum',
+      value: ['start', 'end']
+    },
+    defaultValue: 'end',
+    control: {
+      type: 'radio'
+    },
+    table: {
+      category: 'Pane',
+      defaultValue: {
+        summary: '"end"'
+      }
+    }
+  },
+  'Pane.positionWhenNarrow': {
+    type: {
+      name: 'enum',
+      value: ['inherit', 'start', 'end']
+    },
+    defaultValue: 'inherit',
+    control: {
+      type: 'radio'
+    },
+    table: {
+      category: 'Pane',
+      defaultValue: {
+        summary: '"inherit"'
+      }
+    }
+  },
+  'Pane.width': {
+    type: {
+      name: 'enum',
+      value: ['small', 'medium', 'large']
+    },
+    defaultValue: 'medium',
+    control: {
+      type: 'radio'
+    },
+    table: {
+      category: 'Pane',
+      defaultValue: {
+        summary: '"medium"'
+      }
+    }
+  },
+  'Pane.divider': {
+    type: {
+      name: 'enum',
+      value: ['none', 'line']
+    },
+    defaultValue: 'none',
+    control: {
+      type: 'radio'
+    },
+    table: {
+      category: 'Pane',
+      defaultValue: {
+        summary: '"none"'
+      }
+    }
+  },
+  'Pane.dividerWhenNarrow': {
+    type: {
+      name: 'enum',
+      value: ['inherit', 'none', 'line', 'filled']
+    },
+    defaultValue: 'inherit',
+    control: {
+      type: 'radio'
+    },
+    table: {
+      category: 'Pane',
+      defaultValue: {
+        summary: '"inherit"'
+      }
+    }
+  },
+  'Show footer?': {
+    type: 'boolean',
+    defaultValue: true,
+    table: {
+      category: 'Footer'
+    }
+  },
+  'Footer.divider': {
+    type: {
+      name: 'enum',
+      value: ['none', 'line']
+    },
+    defaultValue: 'none',
+    control: {
+      type: 'radio'
+    },
+    table: {
+      category: 'Footer',
+      defaultValue: {
+        summary: '"none"'
+      }
+    }
+  },
+  'Footer.dividerWhenNarrow': {
+    type: {
+      name: 'enum',
+      value: ['inherit', 'none', 'line', 'filled']
+    },
+    defaultValue: 'inherit',
+    control: {
+      type: 'radio'
+    },
+    table: {
+      category: 'Footer',
+      defaultValue: {
+        summary: '"inherit"'
+      }
+    }
+  }
+}
+
+Playground.args = {
+  containerWidth: 'xlarge',
+  outerSpacing: 'normal',
+  rowGap: 'normal',
+  columnGap: 'normal'
+}
+
+export const PullRequestPage = () => (
+  <PageLayout>
+    <PageLayout.Header>
+      <Box sx={{display: 'flex', flexDirection: 'column', gap: 3}}>
```

Comment:
If there's only one child here with styles, do we want folks to put this sx prop on `PageLayout.Header` or will that break things and we want to discourage that?

Reply:
It'll break things :/ 

The `sx` prop has always been an escape hatch that people can use at their own risk. The `PageLayout` component and subcomponents are especially risky components to pass the sx prop to because they have a lot of complex styles applied to them.